app/electron-client/src/server.ts

```diff
@@ -78,6 +81,9 @@ const HTTP_STATUS_BAD_REQUEST = 400
 const HTTP_STATUS_NOT_FOUND = 404
 const HTTP_STATUS_INTERNAL_SERVER_ERROR = 500
 
+const PROJECT_WATCHER_CALLBACK_DELAY = 30000
+const PROJECT_WATCHER_CALLBACK_TIMEOUT = 300000
```

Comment:
Uh, I don't like this duplication. But I see that it is older than this PR, so let's keep it.